.github/workflows/system-tests-enterprise.yml

```diff
@@ -133,7 +187,7 @@ jobs:
       run: |
         echo "ui_hash=$( \
```

Comment:
@Tankilevitch why do you need to do that here and on line 207? looks like we calculate mlrun ui twice

Reply:
seems like previously we were querying the ui hash once from development and once from the branch it was triggered from, I think this was done to be able to run system tests on different branches for testing where they didn't had equivalent UI branch as well and then it would use the upstream "development" UI hash instead, but when running on the actual branch (development, periodically) it would be the same because the UI does have that branch.